plugins/cohere/credentials.go

```diff
@@ -0,0 +1,111 @@
+package cohere
+
+import (
+	"context"
+
+	"encoding/json"
+	"fmt"
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func Credentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.Credentials,
+		DocsURL:       sdk.URL("https://docs.cohere.com/reference/config"),
+		ManagementURL: sdk.URL("https://dashboard.cohere.ai/"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Authtoken,
+				MarkdownDescription: "JWT used to authenticate to Cohere stored in disk.",
+				Secret:              true,
+			},
+			{
+				Name:                fieldname.Email,
+				MarkdownDescription: "Email used to authenticate to Cohere.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.URL,
+				MarkdownDescription: "URL of the operator server",
+				Optional:            false,
```

Comment:
Let's remove this, as the default is already `false`.

Reply:
Or should this be true?

I see that the importer and provisioner do not require it.